lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -166,7 +180,7 @@ class TableSyntax extends BlockSyntax {
 
     for (var i = 0; i < row.length && i < alignments.length; i++) {
       if (alignments[i] == null) continue;
-      row[i].attributes['style'] = 'text-align: ${alignments[i]};';
+      row[i].attributes['align'] = '${alignments[i]}';
```

Comment:
Oops, I meant to press on this. Regardless of the spec, I don't think we should change this. We should stick to `<td style="text-align:" and avoid changing to a deprecated HTML attribute.

Reply:
"deprecated" is tricky. This syntax will be supported FOR EVER. If there are zero concerns about it working, sticking to spec will save us a lot of sanity in the future IMHO

Reply:
I guess since GitHub itself uses `<td align="`, 🤷  I guess it works fine.

I was assuming this attribute perhaps had minor differences from `style="text-align:`, like different interaction with other CSS, or inheritance or something, but if that's not the case, let's go with what is specified and what GitHub actually uses.